generated/nisync/nisync.proto

```diff
@@ -18,10 +18,131 @@ import "session.proto";
 service NiSync {
   rpc Init(InitRequest) returns (InitResponse);
   rpc Close(CloseRequest) returns (CloseResponse);
+  rpc SendSoftwareTrigger(SendSoftwareTriggerRequest) returns (SendSoftwareTriggerResponse);
+  rpc ConnectClkTerminals(ConnectClkTerminalsRequest) returns (ConnectClkTerminalsResponse);
+  rpc DisconnectClkTerminals(DisconnectClkTerminalsRequest) returns (DisconnectClkTerminalsResponse);
+  rpc ConnectSWTrigToTerminal(ConnectSWTrigToTerminalRequest) returns (ConnectSWTrigToTerminalResponse);
+  rpc DisconnectSWTrigFromTerminal(DisconnectSWTrigFromTerminalRequest) returns (DisconnectSWTrigFromTerminalResponse);
+  rpc ConnectTrigTerminals(ConnectTrigTerminalsRequest) returns (ConnectTrigTerminalsResponse);
+  rpc DisconnectTrigTerminals(DisconnectTrigTerminalsRequest) returns (DisconnectTrigTerminalsResponse);
+  rpc GetAttributeViInt32(GetAttributeViInt32Request) returns (GetAttributeViInt32Response);
```

Comment:
Note: these are the only added methods in this PR. The other methods are added in https://github.com/ni/grpc-device/pull/169